runtime-builder/build.sh

```diff
@@ -18,24 +18,37 @@
 # Required arguments:
 #   project - GCP project ID for image name.
 #   go_version - Go SDK version to bundle into image.
-#   debian_tag - Debian tag for application base image.
-
-usage() { echo "Usage: $0 <project> <go_version> <debian_tag>"; exit 1; }
 
 set -e
 
-if [ "$#" -ne 3 ]; then
+usage() { echo "Usage: $0 <project> <go_version>"; exit 1; }
+
+debian_digest()
+{
+  local digest="$(gcloud beta container images describe gcr.io/google_appengine/debian8:latest | \
+    grep '^Image:' | cut -d'@' -f2 | grep '^sha256:')"
+
+  # The digest consists a prefix "sha256:", the hash string and a trailing newline character.
+  if [[ "$(echo ${digest} | wc -c)" -ne 72 ]]; then
+    echo "$0: malformed digest got: ${digest}"
```

Comment:
Minor nit.  Perhaps make the error message a bit more descriptive.  Say...

"Error: unable to parse digest of debian8 image: ${digest}"